### DISCUSSION TITLE
Add other currencies for upgrades

### DISCUSSION THREAD

**Discussion by OmerBenGera** [OmerBenGera/WildStacker#127] [Pending Feature Requests]
Add the ability to upgrade spawners using other currencies (exp, for example)

**Comment by Overseer9** [comment]
or using custom item to upgrade spawners,

example scenario:
Skeleton Soul, dropped from skeleton that spawn naturally and killed by the player (currently configurable with loottables),
16x Skeleton Souls are required to upgrade skeleton spawner from tier 1 (deffault) to tier 2.
